.github/workflows/ci-host.yml

```diff
@@ -29,11 +29,7 @@ jobs:
           echo ${{secrets.GITHUB_TOKEN}} | docker login ghcr.io -u ${GHCR_USER} --password-stdin
 
           docker build -t ${IMAGE}:${DATE} -t ${IMAGE}:latest -<<EOF
-          # TODO(lukash) temporarily lock fedora to 35 due to bugged
-          # clang-format (clang-tools-extra-0:14.0.0-1.fc36):
-          # https://github.com/llvm/llvm-project/issues/55260
-          # change back to fedora:latest when fixed
-          FROM fedora:35
+          FROM fedora:latest
```

Comment:
I was wondering whether it's better to have `fedora:latest` or a concrete version, because there tend to be issues when moving to the next version and with latest it's kind of arbitrary, unexpected and usually inconvenient if it breaks.

Reply:
Good point. Thinking about it, having an explicit Fedora version would be clearer. I'll change it.